### ISSUE TITLE
Missing configuration re_table re_disposal_building_group

### ISSUE THREAD

**Issue by sjib** [teksi/wastewater#307]
to adapt: building_group

- [ ]                 # fk_disposal=self.get_pk(row.entsorgungref__REL), # TODO check why not available
Has to be matched to re_table

_Originally posted by @sjib in https://github.com/teksi/wastewater/issues/304#issuecomment-2223736823_

**Comment by domi4484** [comment]
@sjib if I am not missing something, this is a many to many relation. That means that we need a link table like `building_group_disposal` in the TEKSI model in order to implement this. 

```
!!@ comment = "Assoziation"
ASSOCIATION Gebaeudegruppe_EntsorgungAssoc =
  EntsorgungRef  -- {0..*} Entsorgung;
  Gebaeudegruppe_EntsorgungAssocRef -- {0..*} Gebaeudegruppe;
END Gebaeudegruppe_EntsorgungAssoc;
```

**Comment by sjib** [comment]
@domi4484 Yes, it is a many to many relation and the re_table already exists (all many to many relation tables start with re_* (as re_maintenance_event_wastewater_structure):

https://github.com/teksi/wastewater/blob/cb2858ee90ccf573f6eaafabcd3114897ab1e4c2/datamodel/changelogs/0001/03_tww_db_dss.sql#L29

and
https://github.com/teksi/wastewater/blob/cb2858ee90ccf573f6eaafabcd3114897ab1e4c2/datamodel/changelogs/0001/03_tww_db_dss.sql#L36

**Comment by sjib** [comment]
@domi4484 Is this solved now? If yes please link to the pull request and close.

**Comment by domi4484** [comment]
fixed by #357
